### PR TITLE
feature(sct.py): list/delete aws palecement groups

### PR DESCRIPTION
**Pull request by temichus**
extend sct.py 'list-resources', 'clean-resourses'
to support AWS placement groups

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by aleksbykov — APPROVED** [review]
LGTM

File: sdcm/utils/common.py [hunk 1]
```diff
@@ -3158,3 +3159,86 @@ def SoftTimeoutContext(timeout: int, operation: str):  # pylint: disable=invalid
...
+        :param available: get all available placement groups
+        :param group_as_region: if True the results would be grouped into regions
+        :param verbose: if True will log progress information
+
+        :return: instances dict where region is a key
+        """
+    placement_groups = {}
+    aws_regions = [region_name] if region_name else all_aws_regions()
+
+    def get_instances(region):
```
Comment (soyacz) [inline sdcm/utils/common.py:3180]: you mention about 'intances' all around, how instances are to placement groups?

**Review by soyacz — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Comment by temichus** [comment]
> @temichus backport isn't clean.
> 
> I would suggest doing it in a PR together with the usage of it

agree

**Pull request merged**